packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js

```diff
@@ -660,6 +660,7 @@ describe('ReactLazy', () => {
     // Mount
     await Promise.resolve();
     expect(() => {
+      // TODO: what's the expectation here? `toBeDefined`?
```

Comment:
this is still failing lint. maybe just delete the `expect`?

Reply:
yeah, this expect can be removed.

Reply:
Sounds good! I'll leave this alone since you were gonna take a look at the failing tests